Technical-Steering-Committee/proposal.md

```diff
@@ -0,0 +1,89 @@
+
+## Role
+
+The TSC will oversee the technical direction of the Zowe project. Responsibilities include, but are not limited to
+
+- Setting the release schedule, release quality standards, and approving releases.
+- Maintaining the overall project governance and processes.
+- Coordinating the use of project infrastructure, include code repositories, issue tracking, CI/CD environments, and communication tools.
+- Establish a Code of Conduct inline with the Open Mainframe Project and Linux Foundation standards.
+- Maintaining the list of committers.
+- Establish and maintain the overall development processes, including any coding, testing, security, and best practice standards for squads to operate within.
+- Faciliate cross-squad interactions, including cross-squad efficiencies, resource management, and balancing the development focuses between technical debt versus new content delivery.
+- Establish and maintain a lifecycle for Zowe squads, serving as a mentor to newly established squads.
+- Encourage participation and mentorship in wider community events such as hackathons or mentorships.
+- Appointing representatives to work with other open source or open standards communities as needed.
+
+## Composition
+
+The TSC is intended to be a representation of Zowe Committers, as well as other architects, subject matter experts, and members of the extended Zowe community. While any of these individuals may participate in the TSC, there will be a defined group of TSC voting members that will vote on actions of the TSC within the roles of the TSC outlined above.
+
+The initial voting members of the TSC will consist of representatives from the Zowe Leadership Committee (ZLC) as well as the current lead or representative from each squad, as defined below. Where the squad lead is also one of the ZLC representative this position should be filled by another squad SME.
+
+  - Joe Winchester (ZLC representative)
+  - Sean Grady (ZLC representative)
+  - Mark Ackert (ZLC representative)
+  - _TBD_ - (Zowe App Framework/Desktop squad)
+  - Mike Bauer (Zowe CLI Squad)
+  - Fernando Rijo Cedeno (Zowe Explorer squad)
+  - Elliot Jalley (Zowe API Squad)
+  - _TBD_ (Zowe Systems - CI/CD squad)
+  - Nan Nan Li (Zowe Documentation squad)
+  - _TBD_ (Zowe Onboarding squad)
+
+The TSC voting members will have no more than one-third (33%) of voting members affiliated with the same employer. In the case where a TSC voting member has an employment change which violates this requirement, the TSC voting member should be replaced by the body in which thier status is provided.
+
+The TSC voting members may adjust the makeup of the TSC voting members after the initial year with approval of the current Zowe Committers. It is expected that any makeup changes would look to align with the growing technical community structure to ensure equitable representation. Any TSC voting member elections will be facilated by the Open Mainframe Project staff to ensure vendor-neutrality. 
```

Comment:
Big change here is to have composition changes go to the Zowe Committers, as this is more technical community focused.

Reply:
Agreed - I like this change.  Thank you